prep-css-syntax/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Syntax</title>
+</head>
+<body>
+  <h1>This h1 tag has been given "Arial" as a font-family.</h1>
+  <p>I am a p tag which has a font-size of 2rem!!!</p>
+  <h3>This h3 tag has a font color of firebrick.</h3>
+  <h2>This h2 has an "italic" font-style.</h2>
+  <style>
+h1 {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+p {
+  font-size: 2rem;
+}
+
+h3 {
+  color: firebrick;
+}
+
+h2 {
+  font-style: italic;
+}
+
+  </style>
```

Comment:
Hi Ralla! Please move the `<style>` element into the `<head>` of the document like this:

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <title></title>
  <style>

  </style>
</head>
<body>
  
</body>
</html>

```